README.md

```diff
@@ -31,18 +67,42 @@ miracles. Any help from you will be very appreciated!
 News
 ----
 
+- 2017-12-15:
+  - Added support for Select and Do loops.
+  - Added support for 'End Sub' and 0 argument return statements.
+  - Added support for #if constructs.
+  - Each VBA stream is now parsed in a separate thread (up to the # of machine cores).
+- 2017-11-28:
+  - Added parsing for private type declarations.
+  - Report calls to CreateProcessA in final report.
+  - Handle Application.Run() of locally defined methods.
+- 2017-11-23:
+  - Added VBA functions Abs, Fix, Hex, String, CByte, Atn, Dir, RGB, Log, Cos, Exp, Sin, Str, and Val.
+  - Added support for 'Exit Function' operator.
+  - Changed math operators to also work with string representations of integers.
+  - Added a configurable iteration limit on loops.
+- 2017-11-14:
+  - Added support for InStr, Replace, Sgn, Sqr, UBound, LBound, Trim, StrConv, Split, StrReverse, and Int VB functions.
+  - Added support for string character subscripting.
+  - Added support for negative integer literals.
+  - Added support for if-then-else statements.
+  - Added support for Const and initial values for global variable declarations.
+  - Handle assignments of boolean expressions to variables.
+- 2017-11-03:
+  - Added support for Left(), Right(), Array(), and BuiltInDocumentProperties() functions.
+  - Added support for global variables.
+  - Fixed some parse errors.
+  - Added analysis of AutoClose() functions.
 - **2016-09-26 v0.02**: First published version
 - 2015-02-28 v0.01: [First development version](https://twitter.com/decalage2/status/571778745222242305)
 - see changelog in source code for more info.
 
-
-
 Download and Install:
 ---------------------
 
 For now, there is no package on PyPI for automated installation. It must be done manually:
 
-- Download the archive from the repository: https://github.com/decalage2/ViperMonkey/archive/master.zip
+- Download the archive from the repository: https://github.com/kirk-sayre-work/ViperMonkey/archive/master.zip
```

Comment:
Please revert to the main repo URL.

Reply:
Fixed in commit c963b624223cefed44fbb08bf7e70bbca0a6bf54.